Bug: @uppy/angular module not found

### Initial checklist

- [x] I understand this is a bug report and questions should be posted in the [Community Forum](https://community.transloadit.com/)
- [x] I searched [issues](https://github.com/transloadit/uppy/issues?q=is%3Aissue) and couldn’t find anything (or linked relevant results below)

### Link to runnable example

_No response_

### Steps to reproduce

- Create a new Angular (v20) project or use an existing Nx monorepo.
- Install Uppy and its Angular integration:

```
npm install @uppy/core @uppy/dashboard @uppy/angular
```

- In a standalone Angular component, try to import and use Uppy Angular components:

```
import { DashboardComponent, DashboardModalComponent } from '@uppy/angular';

@Component({
  standalone: true,
  imports: [
    DashboardComponent,
    DashboardModalComponent
  ],
  ...
})
export class UploadComponent {}
```

- Build the project using:
```
nx build
# or
ng build
```

### Expected behavior

- The @uppy/angular module should be resolvable and importable without errors.

- TypeScript should recognize all exported components and their typings.

- Angular should compile the module statically without NG1010 or related AOT errors.

- Components like DashboardComponent and DashboardModalComponent should be usable in standalone components as documented.

### Actual behavior

After installing @uppy/core and attempting to use @uppy/angular, I encounter the following TypeScript and Angular compiler errors:

[ERROR] TS2307: Cannot find module '@uppy/angular' or its corresponding type declarations.
```
// packages/components/src/lib/upload/upload.component.ts:13:60
import { DashboardComponent, DashboardModalComponent } from '@uppy/angular';
//                                                       ^^^^^^^^^^^^^^^^^^^

```

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -16,6 +16,8 @@ jobs:
     steps:
       - name: Checkout Repo
         uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
 
       - name: Setup Node.js
         uses: actions/setup-node@v4
@@ -55,6 +57,11 @@ jobs:
         run: |
           sed -i '/npmAuthToken:/d' .yarnrc.yml
 
+      - name: Check if Companion was released
+        id: checkIfCompanionWasReleased
+        run: |
+          git diff --exit-code --quiet HEAD^ -- packages/@uppy/companion/package.json || echo "version=$(jq -r .version < packages/@uppy/companion/package.json)" >> $GITHUB_OUTPUT
+
       - name: Upload `uppy` to CDN
         run: corepack yarn run uploadcdn uppy -- --force
         env:
@@ -73,10 +80,15 @@ jobs:
           sed -i "s|https://releases\.transloadit\.com/uppy/v[0-9]*\.[0-9]*\.[0-9]*/|https://releases.transloadit.com/uppy/v$VERSION/|g" README.md
           sed -i "s|https://releases\.transloadit\.com/uppy/v[0-9]*\.[0-9]*\.[0-9]*/|https://releases.transloadit.com/uppy/v$VERSION/|g" BUNDLE-README.md
 
-      - name: Check if Companion was released
-        id: checkIfCompanionWasReleased
+      - name: Commit README updates
         run: |
-          git diff --exit-code --quiet HEAD^ -- packages/@uppy/companion/package.json || echo "version=$(jq -r .version < packages/@uppy/companion/package.json)" >> $GITHUB_OUTPUT
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+          git add README.md BUNDLE-README.md
+          git commit -m "Update CDN version references to latest release" || exit 0
+          git push
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
   # See also companion-deploy.yml
   docker:

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -11,6 +11,8 @@ jobs:
   release:
     name: Release
     runs-on: ubuntu-latest
+    outputs:
+      companionWasReleased: ${{ steps.checkIfCompanionWasReleased.outputs.version }}
     steps:
       - name: Checkout Repo
         uses: actions/checkout@v4
@@ -30,6 +32,9 @@ jobs:
       - name: Build
         run: corepack yarn build
 
+      - name: '@uppy/angular prepublish'
+        run: corepack yarn workspace @uppy/angular prepublishOnly
+
       - run: |
           echo '' >> .yarnrc.yml
           echo 'npmAuthToken: "${NPM_TOKEN}"' >> .yarnrc.yml
@@ -45,3 +50,69 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Remove npmAuthToken from yarnrc
+        run: |
+          sed -i '/npmAuthToken:/d' .yarnrc.yml
+
+      - name: Upload `uppy` to CDN
+        run: corepack yarn run uploadcdn uppy -- --force
+        env:
+          EDGLY_KEY: ${{secrets.EDGLY_KEY}}
+          EDGLY_SECRET: ${{secrets.EDGLY_SECRET}}
+
+      - name: Upload `@uppy/locales` to CDN if it was released
+        run: corepack yarn run uploadcdn @uppy/locales -- --force
+        env:
+          EDGLY_KEY: ${{secrets.EDGLY_KEY}}
+          EDGLY_SECRET: ${{secrets.EDGLY_SECRET}}
+
+      - name: Update CDN version references
+        run: |
+          VERSION=$(yarn workspace uppy exec npm pkg get version | grep -o '"[0-9]*\.[0-9]*\.[0-9]*"' | tr -d '"')
+          sed -i "s|https://releases\.transloadit\.com/uppy/v[0-9]*\.[0-9]*\.[0-9]*/|https://releases.transloadit.com/uppy/v$VERSION/|g" README.md
+          sed -i "s|https://releases\.transloadit\.com/uppy/v[0-9]*\.[0-9]*\.[0-9]*/|https://releases.transloadit.com/uppy/v$VERSION/|g" BUNDLE-README.md
+
+      - name: Check if Companion was released
+        id: checkIfCompanionWasReleased
+        run: |
+          git diff --exit-code --quiet HEAD^ -- packages/@uppy/companion/package.json || echo "version=$(jq -r .version < packages/@uppy/companion/package.json)" >> $GITHUB_OUTPUT
+
+  # See also companion-deploy.yml
+  docker:
+    name: DockerHub
+    needs: release
+    if: ${{ needs.release.outputs.companionWasReleased }}
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_BUILDKIT: 0
+      COMPOSE_DOCKER_CLI_BUILD: 0
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v4
+      - name: Docker meta
+        id: docker_meta
+        uses: docker/metadata-action@902fa8ec7d6ecbf8d84d538b9b233a880e428804 # v5.7.0
+        with:
+          images: transloadit/companion
+          tags: |
+            type=edge
+            type=semver,pattern={{version}},value=${{ needs.release.outputs.companionWasReleased }}
+            # set latest tag for default branch
+            type=raw,value=latest,enable=true
+      - uses: docker/setup-qemu-action@29109295f81e9208d7d86ff1c6c12d2833863392 # v3.6.0
+      - uses: docker/setup-buildx-action@v3
+      - name: Log in to DockerHub
+        uses: docker/login-action@74a5d142397b4f367a81961eba4e8cd7edddf772 # v3.4.0
+        with:
+          username: ${{secrets.DOCKER_USERNAME}}
+          password: ${{secrets.DOCKER_PASSWORD}}
+      - name: Build and push
+        uses: docker/build-push-action@471d1dc4e07e5cdedd4c2171150001c434f0b7a4 # v6.15.0
+        with:
+          push: true
+          context: .
+          platforms: linux/amd64,linux/arm64
+          file: Dockerfile
+          tags: ${{ steps.docker_meta.outputs.tags }}
+          labels: ${{ steps.docker_meta.outputs.labels }}

